tests/TestCase/Mailer/EmailTest.php

```diff
@@ -2487,6 +2497,23 @@ public function testHeaderCharset()
         $this->assertSame($charset, 'Shift_JIS');
     }
 
+    /**
+     * Test transferEncoding
+     *
+     * @return void
+     */
+    public function testTransferEncoding(){
+        // Test new transfer encoding
+        $this->Email->setTransferEncoding('quoted-printable');
+        $this->assertSame($this->Email->getTransferEncoding(), 'quoted-printable');
```

Comment:
Order is wrong, should be: expected, result.

Reply:
Hum, sorry i fix this in #a006420
But i see, the tests above have the same issue.